src/engine/renderer/tr_image.cpp

```diff
@@ -874,7 +874,22 @@ void R_UploadImage( const byte **dataArray, int numLayers, int numMips, image_t
 		format = GL_DEPTH_STENCIL;
 		internalFormat = GL_DEPTH24_STENCIL8;
 	}
-	else if ( image->bits & ( IF_RGBA16F | IF_RGBA32F | IF_RGBA16 | IF_TWOCOMP16F | IF_TWOCOMP32F | IF_ONECOMP16F | IF_ONECOMP32F ) )
+	else if ( image->bits & IF_RGBA16 )
+	{
+		/* Assume this is an old driver without the extension but the RGBA16 blending is available,
```

Comment:
The comment seems to be accidentally copy-pasted

Reply:
Removed.